Packs/NetscoutAED/Integrations/NetscoutAED/NetscoutAED.py

```diff
@@ -152,6 +155,21 @@ def merge_dicts(dict1: dict, dict2: dict) -> dict:
     return {**dict1, **dict2}
 
 
+def handle_args(demisto_args: dict) -> dict:
```

Comment:
add doc-string (what this function is used for)

Reply:
Done